test/broadcasting.jl

```diff
@@ -0,0 +1,18 @@
+# This file is part of TaylorSeries.jl, MIT licensed
+#
+
+using TaylorSeries
+
+using Test
+
+@testset "Broadcasting for Taylor1 expansions" begin
+    t = Taylor1(Int, 5)
+
+    @test 1.0 .+ t == 1.0 + t
+    @test typeof(1.0 .+ t) == Taylor1{Float64}
+    @test 1.0 .+ [t] == [1.0 + t]
+    @test typeof(1.0 .+ [t]) == Vector{Taylor1{Float64}}
+    @test 1.0 .+ [t, 2t] == [1.0 + t, 1.0 + 2t]
+    @test [1.0,2.0] .+ [t 2t] == [1.0+t 1.0+2t; 2.0+t 2.0+2t]
+    @test [1.0] .+ t == 1.0 + t
```

Comment:
This seems to be different from the behaviour with numbers:
```
julia> [1.0] .+ 1

1-element Array{Float64,1}:
 2.0
```
By analogy it seems that the result should be `[1.0 + t]`.

Reply:
Thanks for checking this! Indeed, you are right. I am trying to find my way in this broadcasting, and it is tricky, so it will go slowly. Hold a bit longer... :-)

Reply:
So to clarify that test `@test [1.0] .+ t == 1.0 + t` should be `@test [1.0] .+ t == [1.0 + t]`

Reply:
Thanks for insisting on this. I agree with both of you that the it should return a `Vector`. For the moment I am trying first to make things work as they did before, so there is no problem with packages that depend on `TaylorSeries.jl`, and then get into this subtleties. But the process is a bit rough for the moment...